src/index.html

```diff
@@ -1,12 +1,344 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="page__content">
+    <header class="menu">
+      <div class="menu__container">
+        <a href="#home" class="menu__logo">
+          <img src="./images/logo.svg" alt="logo">
+        </a>
+
+        <nav class="nav menu__nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link is-active" href="#apple">apple</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#samsung">samsung</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#smartphones">smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#computers"
+                data-qa="nav-hover"
+              >laptops &amp; computers</a>
```

Comment:
the child should start from new line
```suggestion
              >
                laptops &amp; computers
              </a>
```